WorkloadManagementSystem/DB/JobElasticDB.py

```diff
@@ -0,0 +1,138 @@
+""" DIRAC JobDB class is a front-end to the main WMS database containing
+  job definitions and status information. It is used in most of the WMS
+  components
+
+  The following methods are provided for public usage:
+
+  getJobParameters()
+
+  setJobParameter()
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Utilities.ReturnValues import S_OK, S_ERROR
+from DIRAC.Core.Base.ElasticDB import ElasticDB as DB
+from DIRAC.ResourceStatusSystem.Client.SiteStatus import SiteStatus
+from DIRAC import gLogger
+
+#############################################################################
+
+
+class JobDB(DB):
```

Comment:
Please, call this class ElasticJobDB. JobDB already exists as a name and it's better to not repeat.